_kurzdokus/sd-2020/sd-202021-engeln-schaefer.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Funktionserweiterung «Bilder vergleichen» im Cranach Digital Archive
+people:
+    - Lara Engeln
+    - Domenic Schäfer
+teaserimage: sd-202021-teaser-engeln-schaefer.png
+gallery:
+    - sd-202021-galerieMobile-engeln-schaefer.png
+    - sd-202021-sammlungMobile1-engeln-schaefer.png
+    - sd-202021-sammlungDesktop2-engeln-schaefer.png
+    - sd-202021-vergleichMobile1-engeln-schaefer.png
+    - sd-202021-vergleichMobile2-engeln-schaefer.png
+    - sd-202021-vergleichMobile3-engeln-schaefer.png
+    - sd-202021-vergleichDesktop3-engeln-schaefer.png
+styleguide: sd-202021-styleguide-engeln-schaefer.svg
```

Comment:
Bitte auf die Benahmung achten:

```
{nachname1}-{nachname2}-{artefakt}.{ext}
```
also z.B.:

```
engeln-schaefer-teaser.png
```

Auch der Name der Markdown-Datei muss lauten:

```
engeln-schaefer.md
```

Reply:
Dateien sind nun alle umbenannt. Passt das so oder soll ich einen neuen Pull Request machen?

Reply:
nein so ist alles in Ordnung!